CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Exprestive Developer Documentation
+
+#### Setup the development environment
+* install Node.JS
+* `rm -rf node_modules ; npm install`
+
+
+#### Run tests
+* run all tests: `npm test`
+* run unit tests: `npm run unit-tests`
+* run feature tests: `npm run feature-tests`
+
+
+#### Update dependencies
+* check whether updates are available: `npm run update-check`
+* automatically update all dependencies to the latest version: `npm run update`
+* manually update individual dependencies
+  * update [package.json](package.json) to the versions given by update-check
+  * check that the new dependencies work
+
+    ```
+    rm -rf node_modules ; npm i
```

Comment:
You don't need to reinstall everything. If you just run "npm i" and your package.json lists a package name/version thats not installed, it will be installed